<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FlashcardApp</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="main_stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="login_page">
      <layout class="QVBoxLayout" name="login_main_verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="login_form_verticalLayout">
         <property name="leftMargin">
          <number>100</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QLabel" name="login_title_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Welcome to Flashcard App</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="login_input_horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="login_fields_verticalLayout">
             <item>
              <widget class="QLabel" name="login_username_label">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Username</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="login_username_lineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="login_password_label">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Password</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="login_password_lineEdit">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="login_submit_button">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="login_askRegister_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Don't have an account yet?</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="login_goToRegister_button">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="register_page">
      <layout class="QVBoxLayout" name="register_main_verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="register_form_verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="register_header_horizontalLayout">
           <item>
            <widget class="QPushButton" name="register_goBackButton">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>&lt; Back</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="register_header_horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="register_title_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Create Your Account</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="register_content_verticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>100</number>
           </property>
           <property name="rightMargin">
            <number>100</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="register_input_horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="register_fields_verticalLayout">
               <item>
                <widget class="QLabel" name="register_username_label">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Username</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="register_username_lineEdit"/>
               </item>
               <item>
                <widget class="QLabel" name="register_password_label">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Create New Password</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="register_password_lineEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="register_passwordConfirm_label">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Input Password Again</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="register_passwordConfirm_lineEdit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="register_submit_button">
               <property name="text">
                <string>Register</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="register_bottom_verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dashboard_page">
      <layout class="QVBoxLayout" name="dashboard_main_verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="dashboard_content_verticalLayout">
         <property name="leftMargin">
          <number>100</number>
         </property>
         <property name="topMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QLabel" name="dashboard_title_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Welcome to Dashboard</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="dashboard_actions_verticalLayout">
           <item>
            <widget class="QPushButton" name="dashboard_goToReview_button">
             <property name="text">
              <string>Review</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="dashboard_goToMyDecks_button">
             <property name="text">
              <string>My Decks</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="review_page">
      <layout class="QVBoxLayout" name="review_main_verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="review_header_horizontalLayout">
         <item>
          <widget class="QPushButton" name="review_goBackButton">
           <property name="text">
            <string>&lt; Back</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="review_header_horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="review_title_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Review Cards</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="review_cardArea_verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="review_cardDisplay_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>200</height>
            </size>
           </property>
           <property name="text">
            <string>Front Text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="review_showAnswer_button">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Show Answer</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="review_difficulty_horizontalLayout">
           <item>
            <widget class="QPushButton" name="review_easy_button">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Easy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="review_medium_button">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Medium</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="review_hard_button">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Hard</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="myDecks_page">
      <layout class="QVBoxLayout" name="myDecks_main_verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="myDecks_header_horizontalLayout">
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="myDecks_goBackButton">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt; Back</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="myDecks_header_horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="myDecks_content_verticalLayout">
         <item>
          <widget class="QLabel" name="myDecks_title_label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>My Decks</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="myDecks_list_stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="myDecks_noDecks_page">
            <layout class="QVBoxLayout" name="myDecks_noDecks_outer_verticalLayout">
             <item>
              <layout class="QVBoxLayout" name="myDecks_noDecks_verticalLayout">
               <item>
                <widget class="QLabel" name="myDecks_noDecks_info_label">
                 <property name="text">
                  <string>There is no deck available.</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="myDecks_list_page">
            <layout class="QVBoxLayout" name="myDecks_list_main_verticalLayout">
             <item>
              <widget class="QScrollArea" name="myDecks_list_scrollArea">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="myDecks_list_scrollArea_content_widget">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>98</width>
                  <height>28</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="myDecks_list_scrollArea_outer_verticalLayout">
                 <item>
                  <layout class="QVBoxLayout" name="myDecks_list_verticalLayout"/>
                 </item>
                 <item>
                  <spacer name="myDecks_list_verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="myDecks_list_actions_horizontalLayout">
           <item>
            <widget class="QPushButton" name="myDecks_list_create_button">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Create New Deck</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="myDecks_list_import_button">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Import Deck</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="card_list_page">
      <layout class="QVBoxLayout" name="card_list_main_verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="card_list_header_horizontalLayout">
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="card_list_goBackButton">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>&lt; Back to My Decks</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="card_list_header_horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="card_list_deck_name_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">margin-bottom: 10px;</string>
         </property>
         <property name="text">
          <string>Deck: [Deck Name]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="card_list_add_card_button">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Add New Card to this Deck</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="card_list_scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="card_list_scrollArea_content_widget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>566</width>
            <height>425</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="card_list_verticalLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="import_placeholder_page"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
